terraform_files/vsphere/terraform.tfvars.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
this is an example file?
if so I would add more examples for values that usually we don't know how they should look (usually addresses/server urls that can be constructed in many ways)